packages/forms-web-app/src/dynamic-forms/s78-questionnaire/journey.js

```diff
@@ -34,9 +34,14 @@ class S78Journey extends Journey {
 				questions.listedBuildingCheck
 			),
 			new Section('Environmental impact assessment', 'environmental-impact'),
-			new Section('Notifying relevant parties of the application', 'notified').addQuestion(
-				questions.howYouNotifiedPeople
-			),
+			new Section('Notifying people about the application', 'notified')
+				.addQuestion(questions.howYouNotifiedPeople)
+				.addQuestion(questions.uploadSiteNotice)
+				.withCondition(
+					response.answers &&
+						(response.answers[questions.howYouNotifiedPeople.fieldName] == 'site-notice' ||
+							response.answers[questions.howYouNotifiedPeople.fieldName].includes('site-notice'))
```

Comment:
the logic to the right of the && is because a checkbox page answer can either be a string (if one box checked) or an array of strings (if multiple boxes checked) - would it make more sense if the checkbox always returned an array?